xml/schema/Core/EntityFile.xml

```diff
@@ -50,8 +50,14 @@
     <required>true</required>
     <comment>FK to civicrm_file</comment>
     <html>
+      <type>Select</type>
       <label>File</label>
     </html>
+    <pseudoconstant>
+      <table>civicrm_file</table>
+      <keyColumn>id</keyColumn>
+      <labelColumn>id</labelColumn>
+    </pseudoconstant>
```

Comment:
I don't think we should add this psedoconstant. As a general rule, any field that is an FK usually does not also need a pseudoconstant as it's redundant and expensive to cache an entire (potentially very large) table in memory as an option list. Instead I think we should do:

```suggestion
      <type>EntityRef</type>
      <label>File</label>
    </html>
```

Reply:
I've applied the suggestion. However this makes EntityFile unusable in APIv4 explorer. Choosing a file fails with `Entity File has no default label field.`